src/inc/internal/BundleWriterHelper.hpp

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include "AppxPackaging.hpp"
+#include "ComHelper.hpp"
+#include "DirectoryObject.hpp"
+#include "AppxBlockMapWriter.hpp"
+#include "ContentTypeWriter.hpp"
+#include "ZipObjectWriter.hpp"
+#include "AppxPackageInfo.hpp"
+#include "BundleManifestWriter.hpp"
+
+#include <map>
+#include <memory>
+#include <future>
+

```

Comment:
Do you use this?

Reply:
I use the map for std::map<std::string, OptionalBundleInfo> optionalBundles;